### ISSUE TITLE
Fullscreen window has ~16px overscan

### ISSUE THREAD

**Issue by MunyuShizumi** [brave/brave-browser#19038]
## Description
Fullscreen window introduces a 16-or-so pixel overscan on all sides, cutting off parts of the interface (ex. scroll bar in its entirety, roughly 50% of top bar). Brave's F11 fullscreen mode is **not** affected.

Started after package upgrade on Arch Linux today. Can't guarantee it's a Brave issue, but only Brave seems to be affected so far.

## Steps to Reproduce
1. Start Brave
2. Make Brave window fullscreen _(window itself, not Brave's F11!)_

## Actual result:
Outer ~16px is cropped on all sides.

![brave-overscan](https://user-images.githubusercontent.com/61267354/139073769-d6bbc3a1-e93d-412f-994e-ccc9f2d2c760.png)
(Image edited: elements partially cropped horizontally to showcase interesting parts and avoid downscaling)

## Expected result:
Typical Chromium-like UI, not cropped.

## Reproduces how often: 

Every launch so far.

## Brave version (brave://version info)
**Brave**: 1.31.87 Chromium: 95.0.4638.54 (Official Build) (64-bit)
**Revision**: d31a821ec901f68d0d34ccdbaea45b4c86ce543e-refs/branch-heads/4638@{#871}
**OS**: Linux

_Local environment info in Miscellaneous Information on bottom_

## Version/Channel Information:
- Can you reproduce this issue with the **current** release? **Yes**
- Can you reproduce this issue with the **beta** channel? N/A
- Can you reproduce this issue with the **nightly** channel? N/A
_At work currently, will test other builds & repo later._

## Other Additional Information:
- Does the issue resolve itself when **disabling Brave Shields**? **No**
- Does the issue resolve itself when **disabling Brave Rewards**? **No**
- Is the issue reproducible on the **latest version of Chrome**? N/A
_At work currently, will test Chromium later._

### Local environment info
**tl;dr**: Arch Linux, Xserver, Xfce/Xfwm, stable Brave build from AUR, upgraded ~1h ago

**OS**: Arch Linux
**Kernel**: linux 5.14.14.arch1-1
**Display server**: xorg-server 1.20.13-2
**Window manager**: xfwm4 4.16.1-2
**Compositor**: xfwm4 4.16.1-2
**Brave**: (AUR) brave-bin 1:1.31.87-1

**Comment by TheMenko** [comment]
Very anoying issue.
OS: Manjaro
Kernel: 5.10.70-1-MANJARO
Brave: Version 1.32.83 Chromium: 95.0.4638.54 (Official Build) beta (64-bit)

**Comment by mesa123123** [comment]
Same Issue here
OS: Xubuntu
Brave Browser 95.1.31.88

**Comment by gitryder** [comment]
AFAIK, this is happening only on XFCE.

**Also, issue is reproducible on the latest stable version of Google Chrome.**

**Comment by mesa123123** [comment]
> AFAIK, this is happening only on XFCE.
> 
> **Also, issue is reproducible on the latest stable version of Google Chrome.**

Hmm so should go to Chromium then...

**Comment by rabinadk1** [comment]
Fixed using this patch in Arch Linux: https://gitlab.xfce.org/xfce/xfwm4/-/commit/488288f6e35d5d3dc077e87fe2ed83129692f62e

**Comment by gitryder** [comment]
> Fixed using this patch in Arch Linux: https://gitlab.xfce.org/xfce/xfwm4/-/commit/488288f6e35d5d3dc077e87fe2ed83129692f62e

@rabinadk1 So we just have to update xfwm, right? to what version?

**Comment by rabinadk1** [comment]
> > Fixed using this patch in Arch Linux: https://gitlab.xfce.org/xfce/xfwm4/-/commit/488288f6e35d5d3dc077e87fe2ed83129692f62e
> 
> @rabinadk1 So we just have to update xfwm, right? to what version?

I don't know about the version. Arch Linux has included this patch when making the package.

**Comment by addow** [comment]
@gitryder Patch is included since [xfwm4 4.16.1-3](https://github.com/archlinux/svntogit-packages/commit/78ae433d75bfe49ba5445a9d2e2536344b1f4c2a#diff-3e341d2d9c67be01819b25b25d5e53ea3cdf3a38d28846cda85a195eb9b7203a). Patch is working for both Brave and Chromium.

**Comment by grewscant** [comment]
> @gitryder Patch is included since [xfwm4 4.16.1-3](https://github.com/archlinux/svntogit-packages/commit/78ae433d75bfe49ba5445a9d2e2536344b1f4c2a#diff-3e341d2d9c67be01819b25b25d5e53ea3cdf3a38d28846cda85a195eb9b7203a). Patch is working for both Brave and Chromium/Google Chrome.

Ah, no I'm not on Arch. Currently on Debian 11. Will have to wait for the patch to be released, eh?
